css/custom.css

```diff
@@ -353,3 +353,6 @@ button {
         opacity: 1;
     }
 }
+.col-md-8 {
```

Comment:
Not a good way to play with bootstrap grid. Please use classes as needed.

You do not need to add extra style to `col-md-8`.

Also, the logo gets stripped off on small screens:

<img width="670" alt="screen shot 2017-07-07 at 10 56 56 pm" src="https://user-images.githubusercontent.com/8039608/27969290-eb8ea19e-6367-11e7-85e6-c55ed07fa986.png">